### PR TITLE
Switch to a home-rolled prettier action

### PR DESCRIPTION
**Pull request by decentralion**
This way we gain control over which version of prettier is used. It's
also simplier.

Paired with: @wchargin

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,5 +31,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "prettier": "^2.1.1"
```
Comment (wchargin) [inline package.json:36]: Pin exact version. Prettier formatting changes in patch versions.

File: package.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
     "start": "docusaurus start",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
-    "deploy": "docusaurus deploy"
+    "deploy": "docusaurus deploy",
+    "prettier": "prettier --write '**/*.{js,md,yml}'"
```
Comment (wchargin) [inline package.json:10]: Rename to not conflict?

**Comment by wchargin** [comment]
Top-level comment: credit where it's due? :-)

File: package.json (hunk 1 above)
Comment (decentralion) [reply package.json]: Ok, done.

**Pull request merged**